content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: blog
+title: 'SIG Node CI Subproject celebrates two years of test improvements'
+date: 2022-02-09
+---
+
+# SIG Node CI Subproject celebrates two years of test improvements
+
+**Authors:** Sergey Kanzhelev (Google), Elana Hashman (Red Hat)
+
+Ensuring the reliability of SIG Node upstream code is a continuous effort
+that takes a lot of behind-the-scenes effort from many contributors.
+There are frequent releases of Kubernetes, base operating systems,
+container runtimes, and test infrastructure that result in a complex matrix that
+requires attention and steady investment to "keep the lights on."
+Since its organization in May 2020, the SIG Node CI subproject has run a weekly
+meeting, and even the full hour is often not enough to complete triage
+of all bugs, PRs, and issues and discuss all related ongoing work within the SIG.
+
+The Node CI subproject is an approachable first stop to help new contributors
+get started with SIG Node. There is a low barrier to entry for new contributors
+to address high impact bugs and test fixes, although there is a long
+road before contributors can ascend up the entire contributor ladder:
+it took over a year to establish two new approvers for the group.
+The complexity of all the different components that power Kubernetes nodes
+and its test infrastructure requires sustained investment over a long period
+of time for developers to deeply understand the entire system,
+both at a high level and in the lower level details.
+
+We have a number of regular contributors at our meetings. However, our reviewers
+and approvers pool is still small. It is our goal to continue to grow
+contributors in order to ensure a sustainable distribution of work
+that does not just fall to a few key approvers.
+
+It's not always obvious how subprojects within SIGs are formed, operate,
+and work. Each is unique to its sponsoring SIG and tailored to the projects
+that the group is intended to support. As a group that has welcomed many
+first-time SIG Node contributors, we'd like to share some of the details and
+accomplishments over the past two years,
+helping to demystify our inner workings and celebrate the hard work
+of all our dedicated contributors!
+
+## Timeline
+
+***May 2020.*** SIG Node CI group was formed on May 11, 2020 with the goal
+to improve SIG Node CI signal and overall observability with more than
+[30 volunteers](https://docs.google.com/document/d/1fb-ugvgdSVIkkuJ388_nhp2pBTy_4HEVg5848Xy7n5U/edit#bookmark=id.vsb8pqnf4gib)
```

Comment:
Beware the extra risk of defacement sharing an “anyone with the link can suggest” document.

Reply:
fixed permissions, thank you!